quickstart/getZiti.ps1

```diff
@@ -0,0 +1,32 @@
+$latestFromGitHub=(irm https://api.github.com/repos/openziti/ziti/releases/latest)
+$version=($latestFromGitHub.tag_name)
+$zitidl=($latestFromGitHub).assets | where {$_.browser_download_url -Match "ziti-windows.*zip"}
+$downloadUrl=($zitidl.browser_download_url)
+$name=$zitidl.name
+$defaultFolder="$env:USERPROFILE\.ziti\bin"
+$toDir=$(Read-Host "Where folder should be used for ziti? [default: ${defaultfolder}]")
+if($toDir.Trim() -eq "") {
+    $toDir=("${defaultfolder}")
+}
+
+$zipFile="${toDir}\${name}"
+if($(Test-Path -Path $zipFile -PathType Leaf)) {
+    echo "The file has already been downloading. No need to download again"
+} else {
+    echo "The file does not exist"
```

Comment:
As a heads up `echo` is just an alias for `Write-Output`. I prefer to not use echo because if someone has installed a GNU toolkit, echo might not be the echo scripts are looking for.

`Write-Output` has its output pipelined (meaning you can pass it to another command).

There is `Write-Host` which provides output but will not be piped.